### PR TITLE
[SPARK-39195][SQL][FOLLOWUP] Remove flaky test of OutputCommitCoordinator

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
Remove flaky test of OutputCommitCoordinator

### Why are the changes needed?
Remove flaky test after disscuss

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Not need

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
ping @cloud-fan

File: core/src/test/scala/org/apache/spark/scheduler/OutputCommitCoordinatorSuite.scala [hunk 1]
```diff
@@ -144,13 +144,6 @@ class OutputCommitCoordinatorSuite extends SparkFunSuite with BeforeAndAfter {
     assert(tempDir.list().size === 1)
   }
 
-  test("If commit fails, if task is retried it should not be locked, and will succeed.") {
```
Comment (cloud-fan) [inline core/src/test/scala/org/apache/spark/scheduler/OutputCommitCoordinatorSuite.scala:147]: do you know why it is flaky?

Comment (AngersZhuuuu) [reply core/src/test/scala/org/apache/spark/scheduler/OutputCommitCoordinatorSuite.scala:147]: > do you know why it is flaky?

The event in DAGScheduler is not stable,  TaskEnd of attempt 1 sometime's coming earlier then attempt 0

**Review by cloud-fan — APPROVED** [review]

**Review by dongjoon-hyun — COMMENTED** [review]
Could you add the discussion link to the PR description, @AngersZhuuuu ?
In general, we don't remove the test coverage. I guess `ignore` with comment might be better.

**Comment by AngersZhuuuu** [comment]
> Could you add the discussion link to the PR description, @AngersZhuuuu ? In general, we don't remove the test coverage. I guess `ignore` with comment might be better.

Discussed offline and have changed it to ignore

**Comment by HyukjinKwon** [comment]
Merged to master.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Thank you, @AngersZhuuuu , @HyukjinKwon , @cloud-fan .

**Pull request closed without merging**